community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,58 @@
+# Best practices for managing onnx issues
+
+## Enforce templates
+Currently there are no templates for creating issues in onnx. This makes it very difficult for community members to triage issues. Going forward templates shall be added and maintained for ONNX issues.
+
+Following templates shall be added
+* ONNX Bug
```

Comment:
Consider rewording to:
* Troubleshoot an issue
* Ask a question
* Propose a new feature, new or updated operator, or other enhancement

Reply:
Agree with 2nd and 3rd... Instead of Troubleshoot an issue may be Report a Bug?